### PR TITLE
[Dungeon] Maraudon

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.
Missing AI included now as well.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588338017197326500.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 276, `maxdmg` = 357, `DamageModifier` = 1.02 WHERE `entry` = 12237;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 187, `maxdmg` = 248, `DamageModifier` = 1.01 WHERE `entry` = 13282;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `DamageModifier` = 1.01 WHERE `entry` = 13596;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `minrangedmg` = 134, `maxrangedmg` = 198, `DamageModifier` = 1.01 WHERE `entry` = 13601;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 276, `maxdmg` = 356, `DamageModifier` = 1.01 WHERE `entry` = 12258;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `minrangedmg` = 122, `maxrangedmg` = 180, `DamageModifier` = 1.01 WHERE `entry` = 12236;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 284, `maxdmg` = 366, `DamageModifier` = 1.01 WHERE `entry` = 12225;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `DamageModifier` = 1.01 WHERE `entry` = 12203;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338017197326500.sql:46]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `DamageModifier` = 1.01 WHERE `entry` IN (13596, 12203);
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `minrangedmg` = 134, `maxrangedmg` = 198, `DamageModifier` = 1.01 WHERE `entry` = 13601;
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 276, `maxdmg` = 356, `DamageModifier` = 1.01 WHERE `entry` = 12258;
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `minrangedmg` = 122, `maxrangedmg` = 180, `DamageModifier` = 1.01 WHERE `entry` = 12236;
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 284, `maxdmg` = 366, `DamageModifier` = 1.01 WHERE `entry` = 12225;
```

File: data/sql/updates/pending_db_world/rev_1588338017197326500.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+UPDATE `creature_template` SET `mindmg` = 122, `maxdmg` = 180, `DamageModifier` = 1.03 WHERE `entry` = 11790;
+UPDATE `creature_template` SET `mindmg` = 159, `maxdmg` = 218, `DamageModifier` = 1.03 WHERE `entry` = 13743;
+UPDATE `creature_template` SET `mindmg` = 175, `maxdmg` = 241, `DamageModifier` = 1.03 WHERE `entry` = 13599;
+UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 262, `DamageModifier` = 1.03 WHERE `entry` = 12207;
+UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 253, `DamageModifier` = 1.03 WHERE `entry` = 13323;
+UPDATE `creature_template` SET `mindmg` = 167, `maxdmg` = 235, `DamageModifier` = 1.03 WHERE `entry` = 11784;
+UPDATE `creature_template` SET `mindmg` = 172, `maxdmg` = 205, `DamageModifier` = 1.03 WHERE `entry` = 11783;
+UPDATE `creature_template` SET `mindmg` = 142, `maxdmg` = 197, `DamageModifier` = 1.03 WHERE `entry` = 11789;
+UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 262, `DamageModifier` = 1.03 WHERE `entry` = 12206;
+UPDATE `creature_template` SET `mindmg` = 82, `maxdmg` = 98, `DamageModifier` = 1.03 WHERE `entry` = 13456;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338017197326500.sql:34]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 242, `DamageModifier` = 1.03 WHERE `entry` IN (12219, 13533);
UPDATE `creature_template` SET `mindmg` = 178, `maxdmg` = 244, `DamageModifier` = 1.03 WHERE `entry` = 12220;
UPDATE `creature_template` SET `mindmg` = 169, `maxdmg` = 224, `DamageModifier` = 1.03 WHERE `entry` IN (11793, 12221, 11792);
UPDATE `creature_template` SET `mindmg` = 165, `maxdmg` = 223, `DamageModifier` = 1.03 WHERE `entry` = 12222;
UPDATE `creature_template` SET `mindmg` = 122, `maxdmg` = 180, `DamageModifier` = 1.03 WHERE `entry` IN (12218, 11791, 11790);
UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 262, `DamageModifier` = 1.03 WHERE `entry` IN (12224, 12207, 12206);
UPDATE `creature_template` SET `mindmg` = 192, `maxdmg` = 272, `DamageModifier` = 1.03 WHERE `entry` = 12223;
UPDATE `creature_template` SET `mindmg` = 169, `maxdmg` = 218, `DamageModifier` = 1.03 WHERE `entry` = 12217;
UPDATE `creature_template` SET `mindmg` = 165, `maxdmg` = 214, `DamageModifier` = 1.03 WHERE `entry` = 12216;
UPDATE `creature_template` SET `mindmg` = 165, `maxdmg` = 221, `DamageModifier` = 1.03 WHERE `entry` = 13141;
UPDATE `creature_template` SET `mindmg` = 189, `maxdmg` = 214, `DamageModifier` = 1.03 WHERE `entry` = 13142;
UPDATE `creature_template` SET `mindmg` = 159, `maxdmg` = 218, `DamageModifier` = 1.03 WHERE `entry` = 13743;
UPDATE `creature_template` SET `mindmg` = 175, `maxdmg` = 241, `DamageModifier` = 1.03 WHERE `entry` = 13599;
UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 253, `DamageModifier` = 1.03 WHERE `entry` = 13323;
UPDATE `creature_template` SET `mindmg` = 167, `maxdmg` = 235, `DamageModifier` = 1.03 WHERE `entry` = 11784;
UPDATE `creature_template` SET `mindmg` = 172, `maxdmg` = 205, `DamageModifier` = 1.03 WHERE `entry` = 11783;
UPDATE `creature_template` SET `mindmg` = 142, `maxdmg` = 197, `DamageModifier` = 1.03 WHERE `entry` = 11789;
UPDATE `creature_template` SET `mindmg` = 82, `maxdmg` = 98, `DamageModifier` = 1.03 WHERE `entry` = 13456;
```

File: data/sql/updates/pending_db_world/rev_1588338017197326500.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 276, `maxdmg` = 357, `DamageModifier` = 1.02 WHERE `entry` = 12237;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 187, `maxdmg` = 248, `DamageModifier` = 1.01 WHERE `entry` = 13282;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `DamageModifier` = 1.01 WHERE `entry` IN (13596, 12203);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 292, `maxdmg` = 377, `minrangedmg` = 134, `maxrangedmg` = 198, `DamageModifier` = 1.01 WHERE `entry` = 13601;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 276, `maxdmg` = 356, `DamageModifier` = 1.01 WHERE `entry` = 12258;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `minrangedmg` = 122, `maxrangedmg` = 180, `DamageModifier` = 1.01 WHERE `entry` = 12236;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 284, `maxdmg` = 366, `DamageModifier` = 1.01 WHERE `entry` = 12225;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 12201;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338017197326500.sql:39]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 187, `maxdmg` = 248, `DamageModifier` = 1.01 WHERE `entry` = 13282;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 292, `maxdmg` = 377, `DamageModifier` = 1.01 WHERE `entry` IN (13596, 12203);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 292, `maxdmg` = 377, `minrangedmg` = 134, `maxrangedmg` = 198, `DamageModifier` = 1.01 WHERE `entry` = 13601;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 276, `maxdmg` = 356, `DamageModifier` = 1.01 WHERE `entry` = 12258;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 269, `maxdmg` = 346, `minrangedmg` = 122, `maxrangedmg` = 180, `DamageModifier` = 1.01 WHERE `entry` = 12236;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 284, `maxdmg` = 366, `DamageModifier` = 1.01 WHERE `entry` = 12225;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 344, `maxdmg` = 444, `DamageModifier` = 1.01 WHERE `entry` = 12201;
```

**Comment by Geekena** [comment]
damage and spells are ok
not found any problem

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**